src/variorum/variorum.h

```diff
@@ -487,4 +487,8 @@ int variorum_get_node_power_domain_info_json(json_t *get_domain_obj);
 /// @brief Test for memory leaks.
 int variorum_tester(void);
 
+/// @brief To get a quoted variorum version, use QuoteMacro(VARIORUM_VERSION).
```

Comment:
Doxygen picks up these comments to generate API docs.
Please add detailed documentation, eg return type of function (string with version number here).
Please add that it is supported on all architectures. 
The `QuoteMacro` part is internal now, so that should be removed from the comment here.

Also, add the new variorum function to API docs here: https://github.com/LLNL/variorum/blob/dev/src/docs/sphinx/api/print_functions.rst 

See examples of other variorum functions and API docs for what it would look like.